cmd/agent/api/server.go

```diff
@@ -106,11 +112,27 @@ func ServerAddress() *net.TCPAddr {
 	return listener.Addr().(*net.TCPAddr)
 }
 
+// We only want to maintain 1 API and expose an external route to serve the cluster level check runner stats.
+// As we have 2 different tokens for the validation, we need to validate accordingly.
 func validateToken(next http.Handler) http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
-		if err := util.Validate(w, r); err != nil {
-			return
+		path := r.URL.String()
+		var isValid bool
```

Comment:
```suggestion
		isValid := false
```
nit: I find this more explicit

Reply:
true, thanks!